UnitTests/GTMSessionFetcherChunkedUploadTest.m

```diff
@@ -809,6 +809,201 @@ - (void)testBigFileURLSingleChunkedUploadFetchRetry {
   [self removeTemporaryFileURL:bigFileURL];
 }
 
+- (void)testBigFileURLSingleChunkedUploadFetchRetryWithOffset {
```

Comment:
Can we also add a test to trigger cancellation of the upload while it's waiting on a retry?